taskRequests/README.md

```diff
@@ -0,0 +1,15 @@
+# Task Request
+
+Firestore Task Requests collection data model
+
+```
+id: <id>,
+taskId: <id>,
+title: string,
+requestedBy:  [string],
```

Comment:
Change syntax to, as current means a array with a single value of type string. 
```ts
requestedBy: Array<string>,
```

Reply:
`[string]` convention is being used in other models. Also, I think `Array<string>` is a javascript array type, but the data stored in firestore is not a javascript array, so this might mislead others.

Reply:
Data Models are independent of firestore or any other store.
`[string]` I felt confusing as it more of a tuple instead of array.

Reply:
updated the model